app/models/event_user.rb

```diff
@@ -1,6 +1,6 @@
 class EventUser < ActiveRecord::Base
   # TODO Do we need these roles?
-  ROLES = [['Speaker', 'speaker'], ['Submitter', 'submitter'], ['Moderator', 'moderator']]
+  ROLES = [%w[Speaker speaker], %w[Submitter submitter], %w[Moderator moderator]]
```

Comment:
In the documentation of the cop I didn't find an option to enable %w(), can you also check please? 
That said I think that we can be flexible about this,  and use whichever format we prefer each time.